packages/auth/src/core/auth/password_policy_impl.ts

```diff
@@ -62,8 +64,15 @@ export class PasswordPolicyImpl implements PasswordPolicyInternal {
         responseOptions.containsNonAlphanumericCharacter;
     }
 
+    const enforcementStateUnspecified =
+      response.enforcementState === 'ENFORCEMENT_STATE_UNSPECIFIED';
+    this.enforcementState = enforcementStateUnspecified
+      ? 'OFF'
+      : response.enforcementState;
```

Comment:
Please consider using less `=== ? :` format as it reduce the readability.

Besides, is  enforcementStateUnspecified being used anywhere else? Can we do 
```
this.enforcementState = response.enforcementState === 'ENFORCEMENT_STATE_UNSPECIFIED'? 'OFF': response.enforcementstate;
```
instead?

Reply:
I did this on separate lines to make it more readable, but I can just convert this to be an if-else statement if that would be better.

Reply:
Done. I converted it to an if-else statement and removed the const.